data/json/uncraft/armor/torso.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "result": "pants_leather",
```

Comment:
I'm guessing this is a typo

Reply:
Ah, yep, copied it over and forgot to change id. This is what I get for making PRs in the middle of the night and not testing the changes thinking it's too simple to fall apart. Give me a second.

Reply:
This is also a great example of why focused PRs have utility. If this PR touched a bunch of items, it would take me 10-20 times as long to notice an error like this.

Reply:
Yup, that's why I actually applied your suggestion to make more smaller PRs instead of one massive one. This one's just one item because it was a lot of swapping around IDs...